core/modules/system/css/system.theme.css

```diff
@@ -585,6 +585,19 @@ tr:hover del.diffchange {
 }
 
 /**
+ * Hero blocks.
+ */
+ .hero-block {
+   background-size: cover;
+   background-repeat: no-repeat;
+   background-position: top center;
+   padding: 50px 100px;
+   text-align: center;
+   color: #ffffff;
```

Comment:
I take it back, the more I'm thinking about it the more I think it should be in config since it's dependent on the image uploaded and the person that uploads the image may not know CSS.

Thinking the checkbox could be "Inverse text color", in case the default is light text on dark bg, or dark text on light bg?